### PR TITLE
run on new travis env + drop legacy ruby versions

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by domcleal** [comment]
Do you want to also increase/specify the minimum Ruby version to match what's tested (in the gemspec and then in .rubocop.yml)? This would also fix #82.

**Comment by roidelapluie** [comment]
Can we please keep ruby 1.9?

File: .rubocop.yml [hunk 1]
```diff
@@ -1,13 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
 AllCops:
-  TargetRubyVersion: 1.9
+  TargetRubyVersion: 2.1
```
Comment (roidelapluie) [inline .rubocop.yml]: Please keep 2.0

**Comment by bastelfreak** [comment]
I would really love to drop all the EOL versions like ruby2.0 and ruby2.1, I've a bad feeling about supporting them. However, little progress is better than nothing.

**Pull request merged**